HospitalFrontend/src/views/SurveyPreview.vue

```diff
@@ -181,7 +181,7 @@ export default {
             if(sum==0)
                 return 0
 
-            return (rate / sum ) * 100
+            return parseFloat(((rate / sum ) * 100).toFixed(2))
```

Comment:
If you divide two integer numbers as a result you will always get integer. So the result of your code will always end with .00

Reply:
This is not true for javascript.

Reply:
The way this works is: first toFixed() rounds any number, integer or float, to 2 decimals, then parseFloat() removes trailing zeros.